### ISSUE TITLE
RITM result

### ISSUE THREAD

**Issue by codwest** [uncbiag/SimpleClick#19]
Thanks for your contribution! could you even try to test all results from ritm using the code provided by author, I find there are some margins, especially for the SBD dataset, with 0.89 NOC@85  and 1.18 NOC@90.

**Comment by qinliuliuqin** [comment]
Sorry, I don't what you mean. Could you elaborate more on your question? Do you need me to test RITM models?

**Comment by codwest** [comment]
I am sorry, my expression confuses you. I use original code(in the paper "Reviving Iterative Training with Mask Guidance for Interactive Segmentation") provided by author, retraining and testing SBD dataset, get a lower result. I use 'hrnetv2_w18_imagenet_pretrained' pretrained weights to train and test SBD dataset. I find my testing results are lower than ones reported in the original paper. Did you meet the similar problem?

**Comment by codwest** [comment]
it's my fault and i found where is my problem.
